### DISCUSSION TITLE
邮件客户端Evolution无法通过v2ray与服务器连接

### DISCUSSION THREAD

**Discussion by Geek-Joe** [v2fly/v2ray-core#1123] [Q&A]
1. 可以访问外网，但就是邮件客户端不能走v2ray与服务器通信
2. 通过日志可以看到对邮件服务器的访问已经被分配到了对应的输出[sg-proxy]
3. **access.log如下：**
2021/07/05 14:16:50 tcp:127.0.0.1:55936 accepted tcp:imap.nixnet.email:143 [sg-proxy]
2021/07/05 14:16:59 tcp:127.0.0.1:55938 accepted tcp:imap.nixnet.email:143 [sg-proxy]
**error.log如下：**
2021/07/05 14:16:59 [Info] [2339434684] proxy/socks: TCP Connect request to tcp:imap.nixnet.email:143
2021/07/05 14:16:59 [Debug] app/dns: domain imap.nixnet.email will use DNS in order: [DOH//dns.nextdns.io UDP:45.90.28.24:53 UDP:45.90.30.24:53]
2021/07/05 14:16:59 [Debug] app/dns: DOH//dns.nextdns.io cache HIT imap.nixnet.email -> [94.16.121.167]
2021/07/05 14:16:59 [Info] [2339434684] app/dispatcher: taking detour [sg-proxy] for [tcp:imap.nixnet.email:143]
2021/07/05 14:16:59 [Info] [2339434684] common/mux: dispatching request to tcp:imap.nixnet.email:143
2021/07/05 14:16:59 [Info] failed to handler mux client connection > proxy/vmess/outbound: connection ends > context canceled
2021/07/05 14:17:04 [Info] app/dispatcher: taking platform initialized detour [de-proxy] for [tcp:api.v2fly.org:443]
2021/07/05 14:17:04 [Info] common/mux: dispatching request to tcp:api.v2fly.org:443
2021/07/05 14:17:04 [Info] transport/internet/websocket: creating connection to tcp:v2de.yuntiss.pro:48802
2021/07/05 14:17:05 [Info] proxy/vmess/outbound: tunneling request to tcp:v1.mux.cool:9527 via tcp:v2de.yuntiss.pro:48802
2021/07/05 14:17:06 [Info] app/observatory: the outbound de-proxyis alive:1.554594527
2021/07/05 14:17:07 [Info] [2339434684] app/proxyman/inbound: connection ends > proxy/socks: connection ends > proxy/socks: failed to transport all TCP response > io: read/write on closed pipe
3. 邮件客户端还需要什么特别的配置才能用么？

**Comment by JarmoHu** [comment]
配置系统代理socks代理试试
